RPIPs/RPIP-49.md

```diff
@@ -65,13 +65,15 @@ Some form of value capture method will be included in the tokenomics rework pack
 * [RPL Buy & LP](RPIP-50.md) - Use the surplus share to buy RPL and deposit it in a liquidity pool. 
 * Direct the surplus share to vote-eligible Node Operators, proportional to their share of vote-eligible RPL.
 
+For Saturn 1, the share will be directed to vote-eligible NOs. There will be a vote prior to Saturn 1 going live to determine the preferred strategy starting at Saturn 2.
```

Comment:
Technically it's "vote-eligible megapool NOs", but maybe your generic description here is good enough to leave vague since it is still mostly correct?

Reply:
"For Saturn 1, the share will be directed based on vote-eligible RPL staked in megapools."